galsim/randwalk.py

```diff
@@ -47,12 +51,21 @@ class RandomWalk(GSObject):
     Initialization
     --------------
     @param  npoints                 Number of point sources to generate.
-    @param  half_light_radius       Half light radius of the distribution of
-                                    points.  This is the mean half light
-                                    radius produced by an infinite number of
-                                    points.  A single instance will be noisy.
+    @param  half_light_radius       Optional half light radius of the
+                                    distribution of points.  This value is used
+                                    for a Gaussian distribution if an explicit
+                                    profile is not sent. This is the mean half
+                                    light radius produced by an infinite number
+                                    of points.  A single instance will be noisy.
+                                    [default None]
     @param  flux                    Optional total flux in all point sources.
+                                    This value is used for a Gaussian distribution
+                                    if an explicit profile is not sent. 
                                     [default: 1]
+    @param  profile                 Optional profile to use for drawing points.
+                                    If a profile is sent, the half_light_radius
+                                    and flux keywords are ignored.
+                                    [default: Gaussian]
```

Comment:
I think default here should be listed as `None`.  The text explains that this means it will automatically make a Gaussian profile in that case.

Reply:
fixed